cosmo_pol/lookup/lut.py

```diff
@@ -181,6 +181,10 @@ def __init__(self):
          # the step for every axis: x1 - x0
         self.axes_step = []
 
+        # to avoid the array shape check error in set_value_table for axis 'd',
+        # which has different axes value for different 'wc'
+        self.axes_len = []
+
```

Comment:
to avoid the array shape check error in set_value_table for axis 'd', which has different axes value for different 'wc'